### DISCUSSION TITLE
Github's notification has a bug

### DISCUSSION THREAD

**Discussion by karottc** [community/community#53836] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

There is a bug with the unread notifications in the top right corner of GitHub. All my notifications have been read, but there is still a small blue dot indicating unread notifications. When I click on it, there is no content.

**Comment by karottc** [comment]
![QQ20230427-105535](https://user-images.githubusercontent.com/1856354/234748272-1314354d-7821-4083-841f-b7ddde834b54.png)

![image](https://user-images.githubusercontent.com/1856354/234748356-204b4ccb-df6f-4233-a355-e74135e0f807.png)

no content , but always show number "1"

**Comment by datpmt** [comment]
you can try https://github.com/notifications?query=is%3Aunread

**Reply by karottc** [reply]
> you can try https://github.com/notifications?query=is%3Aunread

Clicking on this link will have the same effect.
![image](https://user-images.githubusercontent.com/1856354/234764088-6fac9e2c-f161-4dfe-aa7e-88c46390239f.png)

**Comment by radar-blip** [comment]
Its a known bug (ghost notification). See link below for a probable fix.

https://github.com/orgs/community/discussions/6874#discussioncomment-3593547

**Reply by karottc** [reply, marked as answer]
> Its a known bug (ghost notification). See link below for a probable fix.
> 
> https://github.com/orgs/community/discussions/6874#discussioncomment-3593547

thanks very much.

**Reply by radar-blip** [reply]
You're welcome!

**Comment by gokaygurcan** [comment]
It's still happening, in case anyone wonders: 

![image](https://github.com/community/community/assets/1509413/89264554-d8a4-4fd2-843a-5c4e5b05cd4e)
